packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -20,7 +20,30 @@ const BlockMover = ( {
 	onMoveUp,
 	firstIndex,
 	rootClientId,
+	horizontalDirection,
 } ) => {
+	const firstButtonIcon = horizontalDirection ? arrowLeft : arrowUp;
```

Comment:
Hmmm, what would you say about something like that:

Add this on top of file
```js
const horizontalMover = {
  firstButtonIcon: arrowLeft,
  secondButtonIcon: arrowRight,
  firstButtonHint: __( 'Double tap to move the block to the left' ),
  secondButtonHint:  __( 'Double tap to move the block to the right' ),
  firstBlockTitle: __( 'Move block left' ),
  lastBlockTitle: __( 'Move block right' ),
  firstButtonDirection: 'left',
  secondButtonDirection: 'right',
  location: 'position',
}

const vertivcalMover = {
  firstButtonIcon: arrowUp,
  secondButtonIcon: arrowDown,
  firstButtonHint: __( 'Double tap to move the block up' ),
  secondButtonHint: __( 'Double tap to move the block down' );,
  firstBlockTitle: __( 'Move block up' ),
  lastBlockTitle: __( 'Move block down' ),
  firstButtonDirection: 'up',
  secondButtonDirection: 'down',
  location: 'row',
}
```

And here:
```js
const {
  firstButtonIcon,
  secondButtonIcon,
  firstButtonHint,
  secondButtonHint,
  firstBlockTitle,
  lastBlockTitle,
  firstButtonDirection,
  secondButtonDirection,
  location,
} = horizontalDirection ? horizontalMover : verticalMover;
```

Reply:
yeah, much more clear now, thanks !